src/test/resources/database/books/add-book-for-delete.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO books (id, title, author, price, isbn, description, cover_image)
+VALUES (3, 'SadokKoloHatu', 'Shevchenko', 50, '43623uefhef723', 'Awesome', 'SuperImage');
```

Comment:
```suggestion
VALUES (3, 'SadokKoloHatu', 'Shevchenko', 50, '43623uefhef723', 'Awesome', 'SuperImage');

```
dont forget empty line. Do the same for each .sql